plutus-core/plutus-core/test/debruijn/addInteger.plc

```diff
@@ -0,0 +1,7 @@
+(program 0.1.0
```

Comment:
Overall I'm not a *massive* fan of just picking a selection of the test files that we have and reusing them. I'd rather we wrote tests that we think will actually exercise interesting parts of the system.

Reply:
e.g. I don't believe any of these tests actually show up the bug that you fixed